pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -668,6 +668,7 @@ enum ContainerState {
     CONTAINER_RUNNING = 1;
     CONTAINER_EXITED  = 2;
     CONTAINER_UNKNOWN = 3;
+    CONTAINER_OOMKILLED = 4;
```

Comment:
Let's at least strongly type the Reason field,  leaving it as a string is just not correct to me if that's used to match code in kubelet. 
The current code is my opinion on this particular state, yeah it's exited, but still, it's oom killed.